docs/kane-ai-using-datasets.md

```diff
@@ -0,0 +1,95 @@
+
+<script type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify({
+       "@context": "https://schema.org",
+        "@type": "BreadcrumbList",
+        "itemListElement": [{
+          "@type": "ListItem",
+          "position": 1,
+          "name": "Home",
+          "item": "https://www.lambdatest.com"
+        },{
+          "@type": "ListItem",
+          "position": 2,
+          "name": "Support",
+          "item": "https://www.lambdatest.com/support/docs/"
+        },{
+          "@type": "ListItem",
+          "position": 3,
+          "name": "KaneAI Datasets",
+          "item": "https://www.lambdatest.com/support/docs/kane-ai-using-datasets/"
+        }]
+      })
+    }}
+></script>
+Datasets in KaneAI-LambdaTest allow you to manage and reuse input values for test cases efficiently. This guide explains how to create, edit, and manage these data sets while utilizing parameters to run test cases with a variety of inputs, improving flexibility and reducing manual data entry.
+
+## Step 1: Accessing Parameters and Datasets
+Navigate to the Parameters section within your project and review existing data sets under this section.
+> **Note :** Parameters act as placeholders for data, enabling test cases to use varying inputs.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/datasets/1.png').default} alt="Image" className="doc_img img_center"/>
+
+Every test case which uses parameters in KaneAI automatically generates a default data set during authoring. Example: A default data set might capture initial values like form submissions.
+> Default Datasets are immutable in nature i.e. they cannot be modified.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/datasets/2.png').default} alt="Image" className="doc_img img_center"/>
+
+## Step 2: Creating and Editing Datasets
+To create a dataset, navigate to the Datasets section, click "Create Dataset," and define parameters and values as needed. For existing datasets, access the version history of a default dataset, create a copy, and edit the copied dataset to update or add values seamlessly.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/datasets/3.png').default} alt="Image" className="doc_img img_center"/>
+
+## Step 3: Adding Data to Datasets
+Open the editable Dataset. Click on "**Add a Row After**". Enter your desired values (e.g., 5 or Mike).
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/datasets/4.png').default} alt="Image" className="doc_img img_center"/>
+
+### Autofill with AI
+
+Alternatively, you can use the **Autofill with AI** option to automatically populate data fields. AI generates values based on the parameter's name, saving time and effort.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/datasets/5.png').default} alt="Image" className="doc_img img_center"/>
+
+## Step 4: Importing Data via CSV Files
```

Comment:
This needs to be subsection in "Adding Data to Datasets"

### Importing Data via CSV Files